kahuna/public/js/upload/jobs/required-metadata-editor.js

```diff
@@ -44,12 +46,17 @@ jobs.controller('RequiredMetadataEditorCtrl',
     };
 
     ctrl.metadataSearch = (field, q) => {
-
         return mediaApi.metadataSearch(field,  { q }).then(resource => {
             return resource.data.map(d => d.key);
         });
     };
 
+    // As we make a copy of this, we need to watch it
+    // in case the metadata changes from above.
+    $scope.$watch(() => ctrl.originalMetadata, onValChange(metadata => {
+        ctrl.metadata = metadataFromOriginal(metadata);
+    }));
```

Comment:
Rather than setting it earlier and then watching onValChange, could it just be a single `$watch` that initialises and updates it?